dev_cmds/clean-dev.ts

```diff
@@ -18,7 +16,9 @@ async function main() {
         cwd: devPath,
       },
     });
-  } catch {}
+  } catch {
+    process.stdout.write("Could not unlink devcmd from the development folder!\n");
```

Comment:
is there a reason why you're not using `console.log`? (or in this case, `console.error` or `console.warn`)

Reply:
I thought this was a better practice for subprocesses, but I just found out that console.log just calls process.stdout / stderr etc. under the hood :) changing it